### PR TITLE
add prop to Link for proper page routing

### PR DESCRIPTION
**Pull request by Damola18**
# Link Routing Fix

This PR adds additional prop to Chakra Link component to allow proper routing 
Also fixes activity page not using proper linking in attached issue
 
## Linked issues

closes https://github.com/EveripediaNetwork/issues/issues/2729

### CODE REVIEW

File: src/components/Activity/ActivityCard/index.tsx [hunk 1]
```diff
@@ -27,56 +27,58 @@ const ActivityCard = (props: ActivityBodyCardProps) => {
...
-      <Flex flexGrow={1} mx="auto">
-        <ActivityCardImage
-          title={title}
-          link={link}
-          isNotifSubCard={isNotifSubCard}
-          wikiImgObj={wikiImgObj}
-        />
-        <VStack spacing={2} alignItems="flex-start" width="100%">
-          <ActivityCardTop
+    <Link href={link} as={NextLink} style={{ textDecoration: 'none' }}>
```
Comment (kesar) [inline src/components/Activity/ActivityCard/index.tsx:30]: ensure this wrap doesnt break other links in the cards

File: src/components/Wiki/WikiAccordion/AccordionWidget.tsx [hunk 2]
```diff
@@ -46,7 +47,11 @@ const AccordionAddressTemplate = ({
   hasCopied: boolean
 }) => (
   <HStack>
-    <Link target="_blank" href={`https://etherscan.io/address/${content}`}>
+    <Link
+      as={NextLink}
```
Comment (kesar) [inline src/components/Wiki/WikiAccordion/AccordionWidget.tsx:51]: target blank open in other page... not need router

File: src/components/Wiki/WikiPage/InsightComponents/BrainBot/ChatBot.tsx [hunk 3]
```diff
@@ -81,6 +82,7 @@ const ChatBot = ({ wiki }: { wiki: Wiki }) => {
       </Box>
       <BotChatBox wiki={wiki} />
       <Link
+        as={NextLink}
```
Comment (kesar) [inline src/components/Wiki/WikiPage/InsightComponents/BrainBot/ChatBot.tsx:85]: target="_blank"

File: src/components/Activity/ActivityCard/index.tsx (hunk 1 above)
Comment (Damola18) [reply src/components/Activity/ActivityCard/index.tsx:30]: > ensure this wrap doesnt break other links in the cards

Tested it, it doesn't

**Comment by kesar** [comment]
nvm I guess even it doesnt affect the nextlink for external routes, we can keep it so its the same across all the links 👍🏻 Just test if we need the link over the image now that you wrapped the content and once thats solved we can merge 👍🏻

File: src/components/Activity/ActivityCard/index.tsx (hunk 1 above)
Comment (Damola18) [reply src/components/Activity/ActivityCard/index.tsx:30]: > if you add it at that level then shouldnt you remove it from ActivityCardImage.tsx ? not sure if its needed then

Made changes to all

**Comment by kesar** [comment]
niceee

**Pull request merged**